hummingbot/connector/exchange/btc_markets/btc_markets_user_stream_tracker.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+
+import asyncio
+import logging
+from typing import (
+    Optional,
+    List,
+)
+from hummingbot.core.data_type.user_stream_tracker_data_source import UserStreamTrackerDataSource
+from hummingbot.logger import HummingbotLogger
+from hummingbot.core.data_type.user_stream_tracker import (
+    UserStreamTracker
+)
+from hummingbot.core.utils.async_utils import (
+    safe_ensure_future,
+    safe_gather,
+)
+from hummingbot.connector.exchange.btc_markets.btc_markets_api_user_stream_data_source import \
+    BtcMarketsAPIUserStreamDataSource
+from hummingbot.connector.exchange.btc_markets.btc_markets_auth import BtcMarketsAuth
+from hummingbot.connector.exchange.btc_markets.btc_markets_constants import EXCHANGE_NAME
+
+
+class BtcMarketsUserStreamTracker(UserStreamTracker):
```

Comment:
No need to create a specialized user stream tracker for the connector. Please use instead an instance of `UserStreamTracker` class.

Reply:
https://github.com/hummingbot/hummingbot/blob/master/hummingbot/connector/exchange/crypto_com/crypto_com_user_stream_tracker.py#L19